README.md

```diff
@@ -1,2 +1,11 @@
 # lfc-analysis
-A repository for analysis tools for LFC + torque + Global View
+A repository for analysis tools for LFC + torque + Global View in order to create geographic visualizations of proposal data.  
+
+# Setup Development Environment  
+To set up the development environment use either:
+```
+pip install -r requirements.txt
+
+conda env create -f environment.yml
```

Comment:
We should either use pip or conda -- not both.

Reply:
Replaced with pipenv